src/main/java/io/numaproj/numaflow/common/GrpcServerConfig.java

```diff
@@ -10,4 +11,12 @@
 public class GrpcServerConfig {
```

Comment:
you may want to change the name to `GRPCServerConfig`

Reply:
done